docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,390 @@
+    }
+  ]
+}
+```
+
+The service `name`, `cpus` and `mem` are populated from the `config.json` file. The image is populated from the `resource.json` file. We are using HOST mode of networking to dynamically get a port from the available pool. Read [about Marathon ports](https://mesosphere.github.io/marathon/docs/ports.html) to understand modes in detail.
+
+
+### Step 4 : Testing the package
+Now that you have the package built, we need to make sure everything works as expected before publishing to the community.
+
+
+#### Validation using build script.
+You can execute the script inside the `scripts/build.sh` to make sure all the JSON schema comply to specifications and to install any missing libraries. This script is also executed as a precommit hook.
+
+It may throw some error if there are any unrecognized fields in the package files. Fix those error and re-execute the command until the build is successful.
+
+Now, we can run the Universe server locally to test and install our package.
+
+
+#### Build the local Universe server
+Build the Universe Server Docker image
+```bash
+DOCKER_IMAGE="docker-user-name/universe-server" DOCKER_TAG="time-server" docker/server/build.bash
+```
+
+This will create a Docker image `universe-server:time-server` and `docker/server/target/marathon.json` on your local machine
+
+If you would like to publish the built Docker image, run
+```bash
+DOCKER_IMAGE="docker-user-name/universe-server" DOCKER_TAG="time-server" docker/server/build.bash publish
+```
+
+
+#### Run the local Universe server
+Using the `marathon.json` that is created when building Universe Server we can run a Universe Server in our DC/OS cluster which can then be used to install packages.
+
+Run the following commands inside the `server/target` directory to configure DC/OS to use the custom Universe Server (DC/OS 1.8+):
+
+`dcos marathon app add marathon.json`
+
+
+#### Add the Universe repo to DC/OS cluster:
+Now that you have local Universe server up and running, add this to the cluster instance. You can do this from the GUI or CLI. From the `server/target` directory execute
+
+`dcos package repo add --index=0 dev-universe http://universe.marathon.mesos:8085/repo`
+
+
+#### Install the package
+- You can search for you package using something like:
+
+    `dcos package search time`
+- Once you have found our `time-server` package, you can install it on to your cluster using
+
+    `dcos package install time-server`
+- Install the package and if everything works, you have successfully created a package, tested and deployed it!. You can check if your package is running at
+
+    `dcos marathon app list`
+
+- You can browse your endpoint by going to the cluster dashboard and clicking on Services > time-server, you will be able to see a current running task and you can click on any running task to view the endpoint url.
+
+Now continue to next step to publish your package to the DC/OS community.
+
+
+### Step 5 : Publish the package
```

Comment:
Add a bit to this section explaining that to add a package into the Universe, you'll need to create a PR against the mesosphere/universe repo, `version-3.x` branch, and get it to pass all CI tests and have it reviewed before it can be merged.